test/NuGet.Core.Tests/NuGet.DependencyResolver.Core.Tests/RemoteDependencyWalkerTests.cs

```diff
@@ -1682,7 +1828,7 @@ private void AssertPath<TItem>(GraphNode<TItem> node, params string[] items)
                 node = node.OuterNode;
             }
 
-            Assert.Equal(matches, items);
```

Comment:
`Expected` was swapped with `actual`

Reply:
I'm probably guilty of so many of those :D